lua/rzls/server/lsp.lua

```diff
@@ -41,6 +43,12 @@ local requests = {
                     full = true,
                     legend = rzls_client.server_capabilities.semanticTokensProvider.legend,
                 },
+                -- Same capabilities that rzls provides
```

Comment:
We should build this variable directly from the capabilities before disabling them.

Then return the variable